activerecord/lib/rails/generators/active_record/application_record/application_record_generator.rb

```diff
@@ -16,10 +16,10 @@ def create_application_record
 
         def application_record_file_name
           @application_record_file_name ||= if namespaced?
-                                              "app/models/#{namespaced_path}/application_record.rb"
-                                            else
-                                              "app/models/application_record.rb"
-                                            end
+            "app/models/#{namespaced_path}/application_record.rb"
+          else
+            "app/models/application_record.rb"
+          end
```

Comment:
Hmm. There is no offenses in my local, but offense occurs in Code Climate.

https://codeclimate.com/github/rails/rails/activerecord/lib/rails/generators/active_record/application_record/application_record_generator.rb?from_sha=98360a96&to_sha=629158f3#source

With similar code offense has not occurred.

https://github.com/rails/rails/blob/98360a96cc1e0bb6ab9eb31f421a36439e66eefc/activejob/lib/rails/generators/job/job_generator.rb#L30-L37